openedx/core/djangoapps/schedules/management/commands/setup_models_to_send_test_emails.py

```diff
@@ -27,29 +27,41 @@ class ThreeDayNudgeSchedule(ScheduleFactory):
     """
     A ScheduleFactory that creates a Schedule set up for a 3-day nudge email.
     """
+    # TODO: Remove 'start' once data migration is complete, after removing reference to old field step,
+    #  and use new field in column renames.
     start = factory.Faker('date_time_between', start_date='-3d', end_date='-3d', tzinfo=pytz.UTC)
+    # start_date = factory.Faker('date_time_between', start_date='-3d', end_date='-3d', tzinfo=pytz.UTC)
```

Comment:
Why is the line of code to write the same value into `start_date` commented-out in this section and the three below?

Reply:
Start date is used for sending emails to users here. At the moment new field (start_date) is commented out ,until data migration is run to back fill this field, and will replace the old field(start). 
Also start_date is blank at the moment.

Reply:
But it seems like you're writing the same value into the `start` and `start_date` field everywhere else in the PR. So why not here as well? I realize that the code still reads from the `start` field post-merge of this PR. But the pattern in this file seems different than elsewhere in this PR's changes.

Reply:
This file is test code, if I understand it correctly. So I don't think this point should hold things up if you're confident the commenting-out is correct.